### ISSUE TITLE
Provide matrix page as downloadable CSV file

### ISSUE THREAD

**Issue by jkeenan** [cpan-testers/cpantesters-web#24]
Please classify this issue as "wishlist" or "feature request".  As there are quite a few github.com repositories for the CPANtesters project, please let me know if there is a better repository in which to file this issue.

Consider this page from the matrix:

[matrix for Devel-Git-MultiBisect 0.09 on FreeBSD](http://matrix.cpantesters.org/?dist=Devel-Git-MultiBisect%200.09;os=freebsd;reports=1)

It reports results of one version of one CPAN distribution on one operating system.  Since version in question is the latest version of this distribution, the following URL produces the same page (for the time being):

[matrix for Devel-Git-MultiBisect latest release on FreeBSD](http://matrix.cpantesters.org/?dist=Devel-Git-MultiBisect;os=freebsd;reports=1)

Either way, you see that I have gotten a lot of FAILs for this release.  I would like to be able to explore those FAILs without having to click on each individual link in the "Id" column.  That would be easy if:

* I could click on a link which said: "Download this chart as a CSV file"; and

* The CSV file had one extra column, "URL", which spelled out the URL linked to in the second ("Id") column.

If I had such a CSV file I could then write a program to traverse those FAILs and explore the cause of the FAILs.

Now, if I manually copy from the browser and paste the result into vi, I see that the data is already tab-separated.  So conversion from tab-separated to comma-separated should be easy.  That would leave (a) translating the "Id" into a URL and adding a column; (b) adding a download link as the principal tasks remaining.

Thank you very much.
Jim Keenan

**Comment by eserte** [comment]
Please take a look at https://metacpan.org/release/CPAN-Testers-ParseReport
The provided script `ctgetreports` can download all reports for a given CPAN distribution, and may also try to find the reason for the fails (try the `--solve` option).

The results will also be available on http://analysis.cpantesters.org/solved?distv=Devel-Git-MultiBisect-0.09 , but the analysis page sometimes needs some time to do the calculations (or something is stuck --- @andk is this the case currently?)

**Comment by eserte** [comment]
BTW, probably this is the reason for the Devel-Git-MultiBisect-0.09 fails:
```
    HARNESS_OPTIONS = j2
```
I could reproduce it on my smokers.

**Comment by jkeenan** [comment]
> BTW, probably this is the reason for the Devel-Git-MultiBisect-0.09 fails:
> 
> ```
>     HARNESS_OPTIONS = j2
> ```
> 
> I could reproduce it on my smokers.

Slaven, 

Your smokers (and Andreas's) have so far given me all PASS on this version of this distro.  Is that because you do **not** set `HARNESS_OPTIONS = j2`?  Or is it because you somehow have better isolation between individual smoke-test runs on a given rig?

Thanks for the tip about ctgetreports.  I will look into that.

**Comment by jkeenan** [comment]
> BTW, probably this is the reason for the Devel-Git-MultiBisect-0.09 fails:
> 
> ```
>     HARNESS_OPTIONS = j2
> ```
> 
> I could reproduce it on my smokers.

And I was able to reproduce it in my development environment.  I consolidated 4 test files into one, thereby eliminating the possibility of race conditions when each of them touched the same testing data.  I released a new version of Devel-Git-MultiBisect to CPAN and its [smoke-test reports](http://matrix.cpantesters.org/?dist=Devel-Git-MultiBisect) are looking fine, even on the rigs which were FAILing like crazy.

Slaven and Andreas, thanks once again!
